plugins/node/instrumentation-fs/test/parent.test.ts

```diff
@@ -0,0 +1,164 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import {
+  BasicTracerProvider,
+  InMemorySpanExporter,
+  SimpleSpanProcessor,
+} from '@opentelemetry/sdk-trace-base';
+import Instrumentation from '../src';
+import * as assert from 'assert';
+import type * as FSType from 'fs';
+import type { FsInstrumentationConfig } from '../src/types';
+import * as api from '@opentelemetry/api';
+import { AsyncHooksContextManager } from '@opentelemetry/context-async-hooks';
+
+const supportsPromises = parseInt(process.versions.node.split('.')[0], 10) > 8;
+
+const provider = new BasicTracerProvider();
+const memoryExporter = new InMemorySpanExporter();
+provider.addSpanProcessor(new SimpleSpanProcessor(memoryExporter));
+
+const tracer = provider.getTracer('default');
+
+describe('fs instrumentation: requireParentSpan', () => {
+  let plugin: Instrumentation;
+  let fs: typeof FSType;
+  let context: api.Context;
+  let endRootSpan: () => void;
+  let additionalSpans: number;
+
+  const initializePlugin = (pluginConfig: FsInstrumentationConfig) => {
+    plugin = new Instrumentation();
+    plugin.setTracerProvider(provider);
+    plugin.setConfig(pluginConfig);
+    plugin.enable();
+    fs = require('fs');
+    assert.strictEqual(memoryExporter.getFinishedSpans().length, 0);
+  };
+
+  beforeEach(() => {
+    const contextManager = new AsyncHooksContextManager();
+    api.context.setGlobalContextManager(contextManager.enable());
+  });
+
+  afterEach(() => {
+    plugin.disable();
+    memoryExporter.reset();
+  });
+
+  const createSpanTests = (shouldCreateSpan: boolean) => {
```

Comment:
when reading this, I expected the function to create a span itself which is a bit confusing. How about naming it `expectFsSpan`?